revisions/epoch/fractional.go

```diff
@@ -64,6 +104,7 @@ func (TwoHourly) GetData() Data {
 		"The last PR merge commit of two-hour partition of the day, by UTC commit timestamp on master. E.g., epoch changes at 00:00:00, 00:02:00, etc..",
 		time.Hour * 2,
 		time.Hour * 2,
+		"",
```

Comment:
Should this not get the same warning as four_hourly? Or do you think we should cross this road when we decide to add a 3-hourly epoch, since 6/3=2?